runtime/src/execute_cost_table.rs

```diff
@@ -84,17 +96,34 @@ impl ExecuteCostTable {
         self.table.get(key)
     }
 
+    pub fn initialize(&mut self, key: &Pubkey, cost: &u64) {
```

Comment:
isn't this just `upsert` where `count` is `1` and the entry is vacant? i think this adds unnecessary complexity to the interface. (which i think can be removed with the above suggested changes to `upsert`)

Reply:
Yea, it is a related change. To extend the thought above that `upsert` is where all `cost` logics are, (eg., input data is a function of `program_timing`, cost is a function of input data, validation is a function of input data and result, etc), for the sake of single responsibility, I added this function to mechanically initialize container without business logic. Thinking initialization happens once before gears start turning.